internal_ws/ykcompile/src/lib.rs

```diff
@@ -1440,7 +1450,7 @@ impl TraceCompiler {
                 }
             }
 
-            // Create vector to store the allocated memory pointers in, to be handed to the
+            // Create a vector "of what?" to store the allocated memory pointers in, to be handed to the
```

Comment:
It's `Vec<FrameInfo>` (see the signature of `bh_new_vec`).

Agree that this info could be more accessible.

Reply:
Fixed in db5a334.